proto/storage/virtual_machine.proto

```diff
@@ -25,6 +25,9 @@ message VirtualMachine {
     MISSING_SIGNATURE_VERIFICATION_DATA = 3;
   }
   repeated Note notes = 9;
+
+  int32 vsock_cid = 10;
+  bool running = 11;
```

Comment:
Should we make this searchable?

Also the comment from @clickboo  on the name field.  There is no corresponding entry in `pkg/search/options.go`.  This only works because @kylape is using an in memory store and thus didn't generate the Postgres stores.  Store generation would have failed without a corresponding search field.  

So maybe it would be wise to take all the search tags off for now?   (including `hidden` and `store` as I don't think those do anything anymore)

Reply:
> So maybe it would be wise to take all the search tags off for now?

+1, we can revisit searchability when we re-implement the data store.